test/source/tests/content-script.ts

```diff
@@ -0,0 +1,24 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+import test from 'ava';
+
+import { TestWithBrowser } from '../test';
+import { expect } from 'chai';
+import { Value } from '../core/common';
+
+export const defineContentScriptTests = (testWithBrowser: TestWithBrowser) => {
+  test(
+    'content script test',
+    testWithBrowser(undefined, async (t, browser) => {
+      const authorizationHeader = {
+        Authorization: 'Bearer just-to-load-and-run-content-scripts', // eslint-disable-line @typescript-eslint/naming-convention
+      };
+      const gmailPage = await browser.newMockGmailPage(t, authorizationHeader);
+      await gmailPage.waitAny('@content-script-test-result');
+      const allValues = (await gmailPage.readAll('@content-script-test-result')).map(el => el.innerText);
+      // multiple results appear in Firefox. The test is successful only if all divs have the word 'pass'
+      expect(Value.arr.unique(allValues)).to.eql(['pass']);
```

Comment:
The test content script is called 3 times in Firefox for some unknown reason (while only once in Chrome) when opening the gmail page
![image](https://user-images.githubusercontent.com/23335944/227592969-7c75a4b2-4dbf-4efc-8ed5-7a8c951174ad.png)

Reply:
This is an example of success
![image](https://user-images.githubusercontent.com/23335944/227596485-4b74078c-1f72-429b-8752-dafb6087b271.png)

Reply:
Because of these 2 exceptions?
![image](https://user-images.githubusercontent.com/23335944/227711376-85079dd6-90c1-4512-a2ac-84bc97a41225.png)